sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -412,7 +418,7 @@ class DatasetSuite extends QueryTest with SharedSQLContext {
 }
 
 
-case class ClassData(a: String, b: Int)
+case class ClassData(a: String, b: Integer)
```

Comment:
I need to use null in the newly introduced test case. Int does not support null. Or do you want me to add a new case class and keep the existing one untouched?  Thanks!